content/en/community/contributing/code/core/dev-environment.md

```diff
@@ -25,6 +25,8 @@ To deploy the CHT in production, see either [AWS hosting]({{< relref "hosting/3.
 
 CHT Core development can be done on Linux, macOS, or Windows (using the [Windows Subsystem for Linux (WSL2)](https://learn.microsoft.com/en-us/windows/wsl/install)). This CHT Core developer guide will have you install NodeJS, npm, and CouchDB (via Docker) on your local workstation.
 
+Ensure you have at least 8GB RAM, 50GB Free disk and 4 cores.  However, more is for sure better here - you'll see real speed improvements with more RAM and more cores.
```

Comment:
```suggestion
> [!TIP]
> Ensure you have at least 8GB RAM, 50GB Free disk and 4 cores.  However, more is for sure better here - you'll see real speed improvements with more RAM and more cores.
```

Reply:
yay! being more hextra native like this makes our docs easier to read - don't think this is as nitpick as you suggested!

Reply:
To be fair, this is [GitHub Style formatting](https://docs.github.com/en/contributing/style-guide-and-content-model/style-guide#formatting-alerts) for alerts which [are supported by Hugo](https://imfing.github.io/hextra/docs/guide/markdown/#alerts) in general _(I think_).

Reply:
TIL!